configure.ac

```
@@ -6340,6 +6340,21 @@ AS_VAR_IF([with_readline], [no], [
     # in readline as well as newer editline (April 2023)
     AC_CHECK_TYPES([rl_compdisp_func_t], [], [], [readline_includes])
 
+    # Some editline versions declare rl_startup_hook as taking no args, others
+    # declare it as taking 2.
+    AC_CACHE_CHECK([if rl_startup_hook takes arguments], [ac_cv_readline_rl_startup_hook_takes_args], [
+        AC_COMPILE_IFELSE(
+            [AC_LANG_PROGRAM([readline_includes]
+                [[extern int test_hook_func(const char *text, int state);]],
+                [[rl_startup_hook=test_hook_func;]])],
```

Remove unneeded brackets.

```suggestion
                [extern int test_hook_func(const char *text, int state);],
                [rl_startup_hook=test_hook_func;])],
```

---

I agree less quoting is not going to cause problems here, I was just following the rule of thumb to use double quoting for all C source since it may sometimes contain brackets.

---

I'm just nitpicking; feel free to ignore this particular suggestion.